Automattic/newspack-plugin
# Update SEO wizard design

### All Submissions:






### Changes proposed in this Pull Request:

This PR simplifies the SEO interface and fixes a few elements that were visually broken (e.g. ButtonGroup for the title separator)

### How to test the changes in this Pull Request:

1. Navigate to Newspack > SEO
2. Switch to this branch
3. See differences and check if options are still working

### Other information:
